modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -807,6 +807,14 @@ TEST_P(Test_ONNX_layers, CumSumExclusiveInplace)
     testONNXModels("cumsum_exclusive_inplace");
 }
 
+// Issue: https://github.com/opencv/opencv/issues/25363
+// The issue was addressed in 4.x, but the solution does not fit 5.x design
+TEST_P(Test_ONNX_layers, DISABLED_Range)
```

Comment:
I believe we should introduce policy to validate incoming DNN patches on both 4.x and 5.x branches to avoid cases like that.
Testing on both branches should improve proposed solutions.